src/Illuminate/Http/composer.json

```diff
@@ -32,6 +31,9 @@
             "dev-master": "5.7-dev"
         }
     },
+    "suggest": {
+        "illuminate/container": "Required to use the container (5.7.*)."
```

Comment:
Could you specify what exactly needs the container, like the other suggest things (take a look at the other composer.json files for some inspiration :))

Reply:
Yeah, I try to find suggestion `lluminate/container` from other but can't find, other lluminate only require `container` no `suggest`

Reply:
You'll need to look at how the package is used in this particular case to come up with a description. I only suggested looking at the other composer.json files to give you an example of other messages, so you know how to style it.